extensions/binding/org.eclipse.smarthome.binding.bluetooth.yeelightblue/ESH-INF/thing/yeelight_blue2.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bluetooth" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="yeelight_blue2">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bluegiga"/>
```

Comment:
This hurts from the respects of the static code check:

```
[WARNING] .binding.miflora\ESH-INF\thing\miflora.xml:[0]
Missing the supported bridge with id bluegiga
```

Reply:
@martinvw Note that the yeelight binding is not `o.esh.binding.yeelight`, but `o.esh.binding.bluetooth.yeelight` - the same should be done for miflora, i.e. it would just be an extension bundle to the bluetooth binding - and as such have the "bluetooth" binding id.
But in any case you are right: "bluegiga" is not known in general, we will have to define an abstract bridge type that can be referenced by any extension, while having multiple implementations for it. This is not yet supported by the framework.

Reply:
Then we have one more challenge, the milfora binding strived to wrap both an mqtt connector and a bluetooth connector in a single binding (which feels good imho) in the end it would be great of that would be possible somehow...

Reply:
> `o.esh.binding.bluetooth.yeelight`

But how can it be like that in openHAB...

Reply:
Ah, that's an interesting question indeed...
Probably best to directly contribute it to ESH :-)

Reply:
@hakan42 wdyt I intended to start from your openHAB miflora binding 

@kaikreuzer what about the other question:

>Then we have one more challenge, the milfora binding strived to wrap both an mqtt connector and a bluetooth connector in a single binding (which feels good imho) in the end it would be great of that would be possible somehow...

Reply:
We would need to be able to reference bridges from another binding in that case - another feature that isn't yet supported by the framework.
But we would anyhow first need the MQTT bridge infrastructure in place as well. The [according PR](https://github.com/eclipse/smarthome/pull/3839) looks a bit stalled since September :-(.

Reply:
@martinvw , feel free to take over the binding, I'd love if it started getting some proper love. Unfortunately, I am extremely swamped with work at the moment, so I won't be able to do much myself.